### ISSUE TITLE
3.2/develop fails to work

### ISSUE THREAD

**Issue by paulchubatyy** [kohana-minion/tasks-migrations#30]
First of all, I would like to thank you for an awesome module. That's what I was looking for a long time.

Basically 17 of 30 tests fail on 3.2/develop branch.

Most of the issues are caused because of this error:

```
Database_Exception: SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'group' at line 1 [ SELECT * FROM (SELECT *, CONCAT(`group`, ":", CAST(`timestamp` AS CHAR)) AS `id` FROM test_minion_migrations WHERE applied > 0 ORDER BY timestamp DESC) AS temp_table GROUP BY group ]
```

I think the problem is that you are using _GROUP BY group_, which is kind of weird. Maybe the column should be renamed to group_name or something?

It's not recommended to name to columns with the keywords of the sql, AFAIK, so you should avoid columns like description, user, group, etc.

Regards,
Paul Chubatyy

**Comment by loonies** [comment]
in that case, table/column names just have to be quoted with backticks (`)

**Comment by paulchubatyy** [comment]
Yeah, that's what I'm trying fix right now. Maybe that will work. If something comes out I will add the pull request to this issue.

**Comment by kimberv** [comment]
I ran into this issue as well. Explicitly backticking wouldn't be good for cross db support (I use postgres). These were my changes to resolve things if you want them - https://github.com/kimberv/tasks-migrations/commit/9e644c4daf2cff87294865de763f78e1b722b1e1

**Comment by zeelot** [comment]
So I think this was a bug in Kohana `Database` but it's since been fixed (in 3.1 master and on)… let me know if I misunderstood.
